libs/sdk/package.json

```
@@ -38,6 +38,7 @@
     "zod": "^3.25.76"
   },
   "dependencies": {
+    "vectoriadb": "*",
```

Biome (2.1.2)

[error] The key vectoriadb was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)